src/Module.zig

```diff
@@ -3420,9 +3412,18 @@ pub fn semaFile(mod: *Module, file: *File) SemaError!void {
     const struct_ty = sema.getStructType(
         new_decl_index,
         new_namespace_index,
+        null,
         try mod.intern_pool.trackZir(gpa, file, .main_struct_inst),
     ) catch |err| switch (err) {
         error.OutOfMemory => return error.OutOfMemory,
+        // The following errors are from resolving capture values, but the root
+        // struct of a file has no captures.
+        error.AnalysisFail,
+        error.NeededSourceLocation,
+        error.GenericPoison,
+        error.ComptimeReturn,
+        error.ComptimeBreak,
+        => unreachable,
```

Comment:
General zig tip:

```zig
    error.AnalysisFail         => unreachable,
    error.NeededSourceLocation => unreachable,
    error.GenericPoison        => unreachable,
    error.ComptimeReturn       => unreachable,
    error.ComptimeBreak        => unreachable,
```

In the case that one of these is triggered in a debug build it's handy when the cursor points at which one was triggered.

Reply:
Oh, good tip!